src/libjpeg-turbo-2-install-DLL-to-bin-libjpeg-turbo.patch

```diff
@@ -0,0 +1,28 @@
+This file is part of MXE.
+See index.html for further information.
+
+From 703326874bf235adb392b9748ff0a56a08991690 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Sun, 6 Sep 2015 17:45:40 +0100
+Subject: [PATCH] install DLL to bin/libjpeg-turbo not to lib/bin
+
+---
+ ltmain.sh | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/ltmain.sh b/ltmain.sh
+index a968835..6acc338 100644
+--- a/ltmain.sh
++++ b/ltmain.sh
+@@ -5229,7 +5229,7 @@ fi\
+ 	  # place dlname in correct position for cygwin
+ 	  tdlname=$dlname
+ 	  case $host,$output,$installed,$module,$dlname in
+-	    *cygwin*,*lai,yes,no,*.dll | *mingw*,*lai,yes,no,*.dll) tdlname=../bin/$dlname ;;
++	    *cygwin*,*lai,yes,no,*.dll | *mingw*,*lai,yes,no,*.dll) tdlname=../../bin/libjpeg-turbo/$dlname ;;
```

Comment:
> At least http://mxe.redjohn.tk/repos/apt/debian/pool/main/m/mxe-i686-w64-mingw32.shared-jpeg/mxe-i686-w64-mingw32.shared-jpeg_9a_amd64.deb doesn't have the file you are describing.

I mean this could happen. Package `jpeg` has file `i686-w64-mingw32.shared/bin/libjpeg-9.dll`. If I understand correctly, this `9` is something like SONAME in Linux. `libjpeg-turbo` has file `libjpeg-62.dll`. If `9` becomes `62`, the file would be overwritten.

> dynamically linked exes can be directly executed if the DLLs are in the same directory as they are

Good reason.

I don't know where the right place for `libjpeg-62.dll` is.

BTW, jack installs DLLs to `bin/jack`. It may be related to e6b645eaaa3f28527993eb544ec4666c06fe40a9 though it changes `/lib/` to `/lib/jack`. Maybe Jack's build system is too clever to decide to install DLLs to `bin/jack`... I don't know.